config/settings.js

```diff
@@ -51,11 +67,30 @@ window.spinnakerSettings = {
         region: cfDefaultRegion
       },
     }
+    titan: {
+      defaults: {
+        account: titanPrimaryAccount,
+        region: titanDefaultRegion
+      },
+    },
+    kubernetes: {
+      defaults: {
+        account: kubernetesPrimaryAccount,
+        namespace: kubernetesDefaultNamespace
+      },
+    }
+  },
+  whatsNew: {
```

Comment:
what's this? Both the whatsNew container and the gistId.
Does this settings.js file need to change with every new release?
That doesnt seem desirable.